media-libs/t1lib/t1lib-5.1.2-r3.ebuild

```diff
@@ -67,7 +67,7 @@ src_install() {
 	default
 
 	if ! use static-libs; then
-		find "${ED}"usr/lib* -name '*.la' -o -name '*.a' -delete || die
+		find "${ED}"/usr/lib64/ -name '*.la' -o -name '*.a' -delete || die
```

Comment:
Ah, I remember seeing this in `pkgcheck scan` (just about the trailing slash, though) but I must've forgot to rerun it since I had reverted the edit to narrow down why it was failing to build.